checker/src/org/checkerframework/checker/index/upperbound/OffsetEquation.java

```diff
@@ -420,9 +420,9 @@ public static OffsetEquation createOffsetFromNodesValue(
         if (node.getTree() != null && TreeUtils.isExpressionTree(node.getTree())) {
             Long i;
             if (op == '+') {
-                i = IndexUtil.getMinValue(node.getTree(), factory.getValueAnnotatedTypeFactory());
+                i = IndexUtil.getExactValue(node.getTree(), factory.getValueAnnotatedTypeFactory());
```

Comment:
I don't think using min/max will give much more precision anyways.  I simplified the code a bit.